charts/kube-starrocks/values.yaml

```diff
@@ -19,6 +19,7 @@ starrocksOperator:
     # image sliced by "repository:tag"
     repository: starrocks/operator
     tag: latest
+  imagePullPolicy: Always
```

Comment:
be noticed that we are using `latest` tag for the operator, if changed to `IfNotPresent`, it is possible that the image on the host is `latest` but not the latest one on docker hub.

Reply:
In some cases, being able to run is more important than using the latest version。Our testing environment is closed, relying on the original configuration and unable to run normally. Expose parameters to those in need, so that people in situations like mine can operate normally.

Reply:
sure. of course.

Reply:
we shall avoid using `latest` tag in production deployment